From sameo@linux.intel.com  Tue Jun  2 14:53:55 2009
From: Samuel Ortiz <sameo@linux.intel.com>
Date: Wed, 27 May 2009 00:49:33 +0200
Subject: firmware: tuners/xc2028: prepare for FIRMWARE_NAME_MAX removal
To: Greg Kroah-Hartmann <greg@kroah.com>, Andrew Morton <akpm@linux-foundation.org>, Michel Ludwig <michel.ludwig@gmail.com>
Cc: Kay Sievers <kay.sievers@vrfy.org>, Samuel Ortiz <sameo@linux.intel.com>, Mauro Carvalho Chehab <mchehab@infradead.org>
Message-ID: <20090526225134.148561079@linux.intel.com>
Content-Disposition: inline; filename=firmware-xc202.patch


From: Samuel Ortiz <sameo@linux.intel.com>

We're going to remove the FIRMWARE_NAME_MAX definition in order to avoid any
firmware name length restriction.
This patch gets rid of the xc2028 FIRMWARE_NAME_MAX reference.

Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
Cc: Mauro Carvalho Chehab <mchehab@infradead.org>
Cc: Michel Ludwig <michel.ludwig@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/media/common/tuners/tuner-xc2028.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/common/tuners/tuner-xc2028.c
+++ b/drivers/media/common/tuners/tuner-xc2028.c
@@ -48,7 +48,7 @@ MODULE_PARM_DESC(audio_std,
 	"NICAM/A\n"
 	"NICAM/B\n");
 
-static char firmware_name[FIRMWARE_NAME_MAX];
+static char firmware_name[30];
 module_param_string(firmware_name, firmware_name, sizeof(firmware_name), 0);
 MODULE_PARM_DESC(firmware_name, "Firmware file name. Allows overriding the "
 				"default firmware name\n");
